### ISSUE TITLE
Icons disappear on hover

### ISSUE THREAD

**Issue by jagDanJu** [TablePlus/TablePlus-Windows#7]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgreSQL 10.5

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
6

3. The steps to reproduce this issue:
In all dialogues, all icons disappears due to the hover effect.

![image](https://user-images.githubusercontent.com/3613218/45556805-6a8d6500-b83c-11e8-949a-8200ed9409a7.png)

![image](https://user-images.githubusercontent.com/3613218/45556833-8264e900-b83c-11e8-9996-e89f5e5e88d8.png)

**Comment by huyphams** [comment]
Yup, I want to improve it. It's the default behavior of WPF button. I' still finding the way to custom it.

**Comment by jagDanJu** [comment]
@huyphams It Works, Thanks!
